app/helpers/entry_helper.rb

```diff
@@ -0,0 +1,2 @@
+module EntryHelper
```

Comment:
Was this scaffolded in?

Reply:
Yes it was, is that a bad thing to do?

Reply:
Nope, just wanted to know.  It's fine to leave it there for now.  Also explains a lot of the additional stuff being created (cuz if you scaffolded entries, you build a lot of things).